1. Basic Web Development/script.js

```diff
@@ -59,10 +59,9 @@ formSubmission = (form) => {
         const send = document.getElementById('send').checked;
         const read = document.getElementById('read').checked;
 
-        const message = `{"data": "${data}", "record": "${record}", "harvest": "${harvest}", "laugh": "${laugh}", "send": "${send}", "read": "${read}"}`
+        const message = new Permissions(data, record, harvest, laugh, send, read);  
```

Comment:
@dochiasebastian is this class really needed? 😃

Reply:
It is not, I could have just created an object in the stringify method like this `{data, record, harvest, laugh, send, read}`. However, I was thinking that maybe later I might want to do something more with that data so a class might be needed. 
But as you were saying, I shouldn't write code that might be needed later so I will remove it for now.